www/js/reset_password.js

```diff
@@ -1,10 +1,17 @@
-$$(document).on('page:init', '.page[data-name="reset-password"]', function (page) {
+$$(document).on('page:init', '.page[data-name="reset-password"]', function (pageEvent) {
+  const page = pageEvent.detail;
+  const head = $(page.el).find('.page-content');
+
+  const resetPwBtn = head.find('.reset-password-btn');
+  const form = head.find('#reset-password-form');
+  const formInput = form.find('input');
+  const footer = head.find('.reset-password-footer');
```

Comment:
We need to do this more often instead of writing the jquery inline. It is quite expensive.
Also, searching through the entire DOM will be slower and slower as the app grows.

Reply:
Therefore, we should use find to minimize the tree